doc/website/getting-started/installation.md

```diff
@@ -54,6 +53,16 @@ sudo apt install gcc g++ cmake libacl1-dev libncurses5-dev pkg-config
 
 Additionally, there is an optional dependency to the MIT licensed [cpptoml](https://github.com/skystrife/cpptoml) library, which is used to parse a RouDi config file for the mempool config.
 
+### QNX
+
+QNX SDP 7.0 and 7.1 are supported (shipping with gcc 5.4 and gcc 8.3 respectively).
```

Comment:
Is it verfied that QCC/GCC 8.3 builds iceoryx? Otherwise I'd just mention 5.4.

We could add QNX 7.1 support for v1.x or even backport it v1.0 if needed.

Reply:
Hoped to get confirmation here 😉 
7.1 removed!

Reply:
@mossmaurice yes that is verified